.github/workflows/test.yaml

```diff
@@ -1,19 +1,32 @@
-name: Validate SDK
+name: Test SDK
 
 on:
   schedule:
     - cron: "0 4 * * *"
   pull_request:
+  workflow_dispatch:
 
 jobs:
   deploy:
-    name: Validate
-    runs-on: ubuntu-latest
+    name: Test
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-2016, windows-2019, ubuntu-18.04, ubuntu-20.04]
+        python-version: [ 3.6, 3.7, 3.8, 3.9 ]
     steps:
     - name: Checkout
       uses: actions/checkout@v2
-    - name: Validate
-      uses: ./.github/actions/validate
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install Packages
+      run: pip install -r requirements.txt
```

Comment:
just curious, why is -r needed here?

Reply:
Tells pip to [install from a file](https://pip.pypa.io/en/latest/cli/pip_install/#cmdoption-r)